R/accessory.R

```diff
@@ -927,10 +930,12 @@ read_contrasts <-
 
     success <- checkListIsSubset(contrasts$variable, colnames(samples), "contrast variables", "sample metadata")
 
-    blocking <- unlist(lapply(contrasts$blocking, function(x) simpleSplit(x, ";")))
-    blocking <- blocking[!is.na(blocking)]
-    if (length(blocking > 0)) {
-      success <- checkListIsSubset(blocking, colnames(samples), "blocking variables", "sample metadata")
+    if (! blocking_column %in% names(contrasts)){
```

Comment:
Sorry, after my later comment this conditional is unnecessary, since the blocking will always be there :-).

Sorry, I'm doing too many things at once!

Reply:
No problem!